### PR TITLE
VN-524 miner download - messages about failture and retry

### PR DESCRIPTION
**Pull request by huy2nhan**

### CODE REVIEW

**Comment by huy2nhan** [comment]
@anyTV/freedom-vn-team please review

File: zh/index.json [hunk 1]
```diff
@@ -132,7 +134,9 @@
       "add_miners_folder_to_exclusion_list_error": "將採礦工具文件夾添加到排除列表時失敗，請再試一次或手動執行",
       "windows_defender": "Windows Defender",
       "other_virus_software": "其他的防毒軟體"
-    }
+    },
+    "missing_mining_tool": "For the mining with best profit, please try to update the mining tool(s) as soon as possible.",
+    "mining_tool_uptodate": "All mining tools are up-to-date"
```
Comment (pandafox) [inline zh/index.json]: Typo
```
    "error_interruption": "The download is interrupted, please check your network and try again",
    "missing_mining_tool": "For the mining with the highest profit, please try to update the mining tool(s) as soon as possible.",
```

```
    "cancel": "取消"
    "error_interruption": "下載已中斷，請檢查您的網絡，然後重試",
    "error_missing_mining_tool": "為了達到最高利潤的採礦，請盡快更新採礦工具。",
    "mining_tool_uptodate": "所有採礦工具都是最新的"
```

**Comment by huy2nhan** [comment]
@anyTV/freedom-vn-team please review

**Comment by huy2nhan** [comment]
@pandafox please review

**Review by pandafox — APPROVED** [review]

**Pull request merged**